dotCMS/src/integration-test/java/com/dotcms/util/MimeTypeUtilsTest.java

```diff
@@ -25,6 +25,105 @@ public static void prepare() throws Exception {
         IntegrationTestInitService.getInstance().init();
     }
 
+    @Test
+    public void test_match_MimeType_all() throws IOException {
+
+        boolean match = MimeTypeUtils.match("*/*", "image/jpeg");
+
+        Assert.assertTrue("* must match with image/jpeg", match);
+
+        match = MimeTypeUtils.match("*/*", "application/pdf");
+
+        Assert.assertTrue("* must match with application/pdf", match);
+
+        match = MimeTypeUtils.match("*/*", "text/plain");
+
+        Assert.assertTrue("* must match with text/plain", match);
+    }
+
+    @Test
+    public void test_match_MimeType_partial_image() throws IOException {
+
+        boolean match = MimeTypeUtils.match("image/*", "image/jpeg");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "image/*" appears 4 times in this file; the first occurrence is on line 47](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5028307)